src_py/sysfont.py

```diff
@@ -436,7 +427,7 @@ def SysFont(name, size, bold=False, italic=False, constructor=None):
     gotbold = gotitalic = False
     fontname = None
     if name:
-        if isinstance(name, (str, bytes, unicode_)):
+        if isinstance(name, (str, bytes)):
             name = name.split(b"," if str != bytes and isinstance(name, bytes) else ",")
```

Comment:
This looks like a py2 compat thing, since then str would equal bytes. Same on line 503 below.

Reply:
I removed it from more places in the same file too. 
c37e1b5  should remove it.